daprdocs/content/en/reference/api/service_invocation_api.md

```diff
@@ -120,5 +134,19 @@ In case you are invoking `mathService` on a different namespace, you can use the
 
 In this URL, `testing` is the namespace that `mathService` is running in.
 
+#### Non-Dapr Endpoint Example
+
+If the `mathService` service was a non-Dapr application, then it could be invoked using service invocation via an `HTTPEndpoint` as well as a fully qualified domain name (FQDN) URL.
```

Comment:
```suggestion
If the `mathService` service was a non-Dapr application, then it could be invoked using service invocation via an `HTTPEndpoint`, as well as a Fully Qualified Domain Name (FQDN) URL.
```